docs/_sidebar.md

```diff
@@ -0,0 +1,20 @@
+- Getting Stated
+    - Core Concepts
```

Comment:
What about moving this and child items be up one level (i.e. "Core Concepts" at the same level as "Usage")?

Reply:
So, I would ditch Getting Started (not anything really hard to explain here really, except the items themselves). 
And then have Core Concepts, Examples and Usage all at same level, is that right ?

Reply:
Yes, that's what I was thinking. It just looked like a heading with no content when I was checking it out. But if you plan to add something to Getting Started, you could leave it in.

Reply:
I like the way you proposed it. Done :)